train.py

```diff
@@ -64,6 +64,11 @@
   flags.DEFINE_integer(
       "batch_size", 1024,
       "How many examples to process per batch for training.")
+  
+  flags.DEFINE_integer(
+    "epochs", 100,
```

Comment:
+1 for num_epochs.

Regarding the default value, I am also OK with a small number, as long as we mention in the docs that training WILL stop after <default value> epochs.